doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md

```diff
@@ -307,6 +307,7 @@ As seen above, the default configuration distributed with the image is tied to c
     * The port for Postgis is `3306` but can be changed through the CYGNUS_POSTGIS_PORT environment variable.
     * The user for Postgis is `postgresql` but can be changed through the CYGNUS_POSTGIS_USER environment variable.
     * The pass for Postgis is `postgresql` but can be changed through the CYGNUS_POSTGIS_PASS environment variable.
+    * CYGNUS_POSTGIS_SKIP_CONF_GENERATION: true skips the generation of the conf files, typically this files will be got from a volume, false generate the conf files.
```

Comment:
Maybe more precise:

false generate the conf files -> false autogenerate the conf files from the rest of environment variables.

Reply:
Ok, the change is [here](https://github.com/telefonicaid/fiware-cygnus/pull/1657/commits/61aef2004d42967c5293839c10bc355144f04747).